arviz/utils/utils.py

```diff
@@ -182,3 +182,15 @@ def _create_flat_names(varname, shape):
     labels = (np.ravel(xs).tolist() for xs in np.indices(shape))
     labels = (map(str, xs) for xs in labels)
     return ['{}__{}'.format(varname, '_'.join(idxs)) for idxs in zip(*labels)]
+
```

Comment:
`scale_text` is already used by  `posteriorplot`. Given that you also need it for `pairplot` I suggest to move  `scale_text` inside `pltos/plot_utils.py` and from there import it into `posteriorplot`  and `pairplot`.